CHANGELOG.md

```diff
@@ -3,7 +3,88 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 0.59.0
+
+This release adds tooling to help prepare existing policies for the upcoming OPA 1.0 release.
+This release also contains a mix of improvements, bugfixes and security fixes for third-party libraries.
+
+### Rego v1
```

Comment:
I'm unsure about this section. Is this the time to inform the community about this, or should we wait until `opa fmt` also attempts to fix deprecated built-in calls?

We need get this out there sometime, though.

Reply:
I think as long as we're clear that "the future release" is not this one, and we add some words such that this is unmistakably clear, it's good to get this out. :+1: